frontend/src/slices/notifications/notification.slice.ts

```diff
@@ -0,0 +1,28 @@
+import { type PayloadAction } from '@reduxjs/toolkit';
+import { createSlice } from '@reduxjs/toolkit';
+
+import { showErrorNotification } from './actions.js';
+
+type State = {
+  message: string;
+};
+
+const initialState: State = {
+  message: '',
+};
+
+const { reducer, actions, name } = createSlice({
+  name: 'notification',
+  initialState,
+  reducers: {},
+  extraReducers: (builder) => {
+    builder.addCase(
+      showErrorNotification,
+      (state, action: PayloadAction<string>) => {
+        state.message = action.payload;
+      },
+    );
+  },
```

Comment:
We don't need to do anything in store with this notification

Reply:
Ok, I have doubts about this one I was thinking that we need to maybe save the state on the store for some future implementations but I don't know the full scope of the error handling system, so should I delete the file?
I'm confused because the other comments say:

let's rename notifications slice to app slice, and method to notify (it will accept not only errors)

so should I eliminate the notification slice (app slice) and not manage states or what should it do?

Reply:
The slice is not necessary if we don't need to manage the state.

`If you have an action that doesn't update the store but still needs to be dispatched, you don't necessarily need to create a separate slice for it. Slices are primarily used to manage state and associated logic within the Redux store. If your action doesn't affect the state, it might be better suited as a standalone action creator outside of a slice.`

What I did was follow the structure of the other slice as a reference but leave the slice empty.